pulsar/pulsar-impl/src/main/java/com/datastax/oss/sink/pulsar/util/kite/package-info.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/** This package is adapted from https://github.com/kite-sdk */
+package com.datastax.oss.sink.pulsar.util.kite;
```

Comment:
Could you elaborate more on why do we need those adapted classes/?

Reply:
it is the tool for inference avro schema from json content. there's similar tool `JsonToAvroSchema` with the same functionality. presently the former is in use. (the latter looks faster, though)